state/migration_internal_test.go

```diff
@@ -368,6 +368,8 @@ func (s *MigrationSuite) TestApplicationDocFields(c *gc.C) {
 		// RelationCount is handled by the number of times the application name
 		// appears in relation endpoints.
 		"RelationCount",
+		// TODO(caas) - add to export/import
```

Comment:
please create a bug and/or card so it doesn't get forgotten

Reply:
created card